configure.ac

```diff
@@ -364,6 +364,21 @@ while test -n "$hname"; do
   test "$hname_tmp" = "$hname" && break
 done
 
+## ----------------------------------------------------------------------
+## Check if we should build the HDF5 API tests
+AC_ARG_ENABLE([api-tests],
+    AS_HELP_STRING([--enable-api-tests], [Enable building of API tests]),
+    [enable_api_tests=$enableval], [enable_api_tests=no])
+
+## ----------------------------------------------------------------------
+## Check if we should build with async API tests
+AC_ARG_ENABLE([async],
+    AS_HELP_STRING([--enable-async], [Enable building of async library and tests]),
```

Comment:
Just to avoid confusion, this should probably be something like `--enable-api-tests-async`